cli/tests/unit/worker_types.ts

```diff
@@ -5,7 +5,7 @@ unitTest(
   { permissions: { read: true } },
   function utimeSyncFileSuccess() {
     const w = new Worker(
-      new URL("../workers/worker_types.ts", import.meta.url).href,
+      new URL("../testdata/workers/worker_types.ts", import.meta.url).href,
```

Comment:
I'm not sure what this test does. It seems like I can put any path here and it will pass.